narayana-release-process.sh

```diff
@@ -143,6 +138,33 @@ then
 fi
 cd ..
 
+# Raising PR for quarkus to upgrade Narayana to current version.
+QRKSISSUE = WFLYISSUE
+if [ ! -d "quarkus" ]
+then
+  (git clone git@github.com:jbosstm/quarkus.git -o jbosstm; cd quarkus; git remote add upstream git@github.com/quarkusio/quarkus.git)
+fi
+cd quarkus
+git fetch jbosstm
+git branch | grep $QRKSISSUE
+if [[ $? != 0 ]]
+then
+  git fetch upstream;
+  git checkout -b ${QRKSISSUE}
+  git reset --hard upstream/master
+  cd /bom/runtime/
```

Comment:
I think this won't work as you go with abs path

Reply:
locally this was working, but anyways it is always better to use full path, so I will change it as requested.